taskcluster/ci/codecov/kind.yml

```diff
@@ -35,6 +35,7 @@ tasks:
                 - codecov-uploader
             tests:
                 - coverage.xml
+        run-on-tasks-for: ["github-push", "github-pull-request"]
```

Comment:
Will this work for `github-pull-request:*` or do we need to `rework run-on-tasks-for`?

Reply:
The `:*` construct is only internal to ci-configuration. Taskcluster Github will either set `tasks_for = github-pull-request` or `tasks_for = github-pull-request-untrusted`. So this will prevent the task from running on the latter without re-working anything.